jumeg_math.py

```diff
@@ -81,4 +79,44 @@ def calc_tkeo(signal):
     tkeo[-1] = 0.
 
     # return results
-    return tkeo
\ No newline at end of file
+    return tkeo
+
+
+##################################################
+#
+# destroy phase/time info on any data array
+#
+##################################################
+def shuffle_data (data_trials, mode='shuffle'):
```

Comment:
Moving to jumeg_utils and changing the name of the function isfine with me. 
I would would avoid naming it something with "randomize..." as there are plenty of functions available; all trying to fit a specific distributuion. Instead we could use something with "desync_trials".
But how about defining two separate functions: "shuffle_data" and "shift_data" in this way we would keep it very intuitive.